Main/src/Coffee/FindMin.java

```diff
@@ -0,0 +1,30 @@
+package Coffee;
+
+public class FindMin {
```

Comment:
@GayaneGulanyan what mean FindMin , maybe this is MachineCalculator ?

Reply:
I need to find minimum from 2-4 arg., I tried to find build in function, but Math function returns min from only 2 args, so 
I decided to write my own function.